engine/src/state.rs

```diff
@@ -123,14 +123,18 @@ impl State {
             .len()
             > 1
         {
-            let pieces = self.board.board.get(&target_position).unwrap();
+            let pieces = self
+                .board
+                .board
```

Comment:
is this really supposed to be `self.board.board`?

Reply:
Yes, `board` is a struct and it has a `member` called `board` not the best naming, but I didn't come up with anything better at that point